custom/enikshay/tests/two_b_datamigration/test_helpers.py

```diff
@@ -48,6 +51,18 @@ def test_empty_value_in_row(self):
         value = MockColumnMapping.get_value("col2", row)
         self.assertEqual(value, None)
 
+    def test_required_value_missing(self):
+        row = self.get_row(["0", None, "2"])
+        # col1 is required
+        with self.assertRaises(Exception) as cm:
```

Comment:
Is `Exception` the most specific exception can be caught here? Seems overly broad, and the test could be passing due something broken causing an entirely unrelated exception to be raised.